src/locales/messages.json

```diff
@@ -9,5 +10,6 @@
   "theme.light": "Light theme",
   "theme.dark": "Dark theme",
   "zoom.in": "Zoom in",
-  "zoom.out": "Zoom out"
+  "zoom.out": "Zoom out",
+  "example.header": "Example test!"
```

Comment:
is this required ?

Reply:
`example.header`

Reply:
@btzr-io It's for demonstration purpose, see `src/example/index.js`